src/Event/MultiversionManagerEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\multiversion\Event;
+
+use Drupal\Core\Entity\ContentEntityTypeInterface;
+use Drupal\multiversion\MultiversionMigrationInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * MultiversionManagerEvent class.
+ */
+class MultiversionManagerEvent extends Event {
+
+  /**
+   * @var \Drupal\Core\Entity\ContentEntityTypeInterface[]
+   */
+  protected $entityTypes;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct($entity_types, MultiversionMigrationInterface $migration) {
+    $this->entityTypes = $entity_types;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityTypes() {
```

Comment:
rename into getEntityType that will consume $entity_type argument and will return NUll or <code>$this->entityTypes[$entity_type]</code>